portfolio/src/main/webapp/scripts/templatemo-script.js

```diff
@@ -46,37 +46,39 @@ $(document).ready(function () {
     document.querySelector('.tm-current-year').textContent = new Date().getFullYear();
 });
 
-var pictures = new Array("images/apollo.jpg", "images/bob.jpeg", "images/boomer.jpg", "images/drift.jpg",
-                        "images/filbert.jpg", "images/kiki.jpg", "images/lolly.jpg", "images/punchy.jpg",
-                        "images/rosie.jpg");
+var pictures = new Array(new Array("images/blathers-negative.png", "images/isabelle-negative.png"), 
```

Comment:
I think you can do it simpler by:
```
const pictures = [["images/blathers-negative.png", "images/isabelle-negative.png"...],
                  ["images/blathers-neutral.png", ....],
                  ...
```

Reply:
lets also move the pictures constant above the modify template. since this is only used there its better to keep the code near for easier code navigating

Reply:
Ok! I've made the change